docs/apache-airflow/concepts/scheduler.rst

```diff
@@ -180,10 +338,43 @@ The following config settings can be used to control aspects of the Scheduler HA
   SchedulerJobs.
 
   This setting controls how a dead scheduler will be noticed and the tasks it
-  was "supervising" get picked up by another scheduler. (The tasks will stay
-  running, so there is no harm in not detecting this for a while.)
+  was "supervising" get picked up by another scheduler. The tasks will stay
+  running, so there is no harm in not detecting this for a while.
 
   When a SchedulerJob is detected as "dead" (as determined by
   :ref:`config:scheduler__scheduler_health_check_threshold`) any running or
   queued tasks that were launched by the dead process will be "adopted" and
   monitored by this scheduler instead.
+
+- :ref:`config:scheduler__dag_dir_list_interval`
+  How often (in seconds) to scan the DAGs directory for new files.
+
+- :ref:`config:scheduler__file_parsing_sort_mode`
+  The scheduler will list and sort the DAG files to decide the parsing order.
+
+- :ref:`config:scheduler__max_tis_per_query`
+  The batch size of queries in the scheduling main loop. If this is too high, SQL query
+  performance may be impacted by one or more of the following:
+
+  - reversion to full table scan - complexity of query predicate
```

Comment:
I don't think this is true -- it looks like it just applies a `LIMIT` to the query.

(i.e. I can't see anywhere the gets TIs and then passes `n` of these to TI.filter_for_tis)

Reply:
I took it from the original description - so I will remove it there too.